admin-dev/themes/new-theme/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+// http://eslint.org/docs/user-guide/configuring
+
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    node: true,
+    es6: true,
+    jquery: true,
+  },
+  globals: {
+    google: true,
+    document: true,
+    navigator: false,
```

Comment:
Just to know, why navigator ?

Reply:
This is something we can have with `window`